Tests/SwinjectAutoregistrationTests/TypeParserTests.swift

```diff
@@ -128,14 +128,6 @@ class TypeParserSpec: QuickSpec {
                 XCTAssertEqual(type, Type.closure(parameters: [], returnType: void, throws: true))
             }
             
-            it("scans implicitly unwrapped optional") {
-                let string = "\((String!).self)"
```

Comment:
Latest Xcode does not allow `(String!).self`. This was producing `"Optional<String>"` anyway, which is the case covered by the below test case.